docker-compose.yml

```diff
@@ -156,18 +200,22 @@ services:
       - "9114:9114"
 
   keycloak:
-    image: quay.io/keycloak/keycloak:16.1.1
+    image: quay.io/keycloak/keycloak:18.0.2
     container_name: keycloak
-    command: ["-b", "0.0.0.0", "-Dkeycloak.migration.action=import", "-Dkeycloak.migration.provider=dir", "-Dkeycloak.migration.dir=/opt/jboss/keycloak/realm-config", "-Dkeycloak.migration.strategy=OVERWRITE_EXISTING", "-Djboss.socket.binding.port-offset=1000", "-Dkeycloak.profile.feature.upload_scripts=enabled"]
+    hostname: keycloak
+    command: [ "start-dev", "--import-realm" ]
+    restart: unless-stopped
```

Comment:
these restart's are awesome...  I never used them...   also love the nice pretty use of `command`

Reply:
I think I'm using your Solr-Chorus work here again ;)